CHANGES.md

```diff
@@ -57,11 +67,22 @@ To be released.
 
  -  `BlockChain<T>.PreloadAsync()` method became to omit rendering of
     `IAction`s in the preloaded behind blocks.  [[#272], [#343]]
+    
  -  `Swarm<T>` became to have two more message types: `GetRecentStates` (`0x0b`)
     and `RecentStates` (`0x0c`).  [[#272], [#343]]
+    
+ - Peers now connected via Kademlia protocol. Peers are now selectively connected to
```

Comment:
Also, it would be better if there is a link about [Kademlia](https://en.wikipedia.org/wiki/Kademlia).